frontend/src/routes/(voters)/questions/+page.svelte

```diff
@@ -1,15 +1,21 @@
 <script lang="ts">
-  import {_} from 'svelte-i18n';
   import {page} from '$app/stores';
 
   const firstQuestionUrl = `${$page.url.pathname.replace(/\/$/, '')}/${$page.data.questions[0].id}`;
+
+  const questionCategories = new Set<string>();
+  $page.data.questions.forEach((question: any) => questionCategories.add(question?.category));
```

Comment:
We could change `any` to `QuestionProps` or leave it out, because `ts` should be able to figure the type out based on `app.d.ts`.

In theory, the categories count will now also include also questions that don't have a categories (i.e. it's `undefined`), although maybe that's all well, because the uncategorized ones are kind of one category :smile:.